compiler/crates/common/src/feature_flags.rs

```diff
@@ -64,6 +64,10 @@ pub struct FeatureFlags {
     /// Create normalization nodes for client edges to client objects
     #[serde(default)]
     pub emit_normalization_nodes_for_client_edges: bool,
+
+    /// Annotate the generated files with jsdoc comments
+    #[serde(default)]
+    pub generate_jsdoc: FeatureFlag,
```

Comment:
If we're not going to use the `is_enabled_for` aspect of FeatureFlag, we should just use a bool here.